docs/en/api/POSIX-API.md

```diff
@@ -344,9 +345,8 @@ characteristics, please be aware that:
 * Files can be written only once, only sequentially, and never be modified.
   That means overriding a file is not allowed, and an explicit combination of delete and then create
   is needed.
-  For example, the `cp` command will fail when the destination file exists.
-  `vi` and `vim` commands will succeed because the underlying system do create, delete, and rename
-  operation combinations.
+  For example, the `cp` command would fail when the destination file exists.
+  `vi` and `vim` commands would only succeed creating a file. Any modification after the file is created would fail.
```

Comment:
After test default Vi editor on CentOS 7, any modification to a created file as long as it needs to be saved, even without any changes, would fail to save. Any operation to the mount point (for example, `ls`), would return error `cannot open directory .: Transport endpoint is not connected`.

Reply:
Different POSIX API has differnet behavior. 
vi/vim may succeed in modifying a file because in the backend the operating system do create a new file, delete the old file, rename the new file back to the original name

Reply:
centos 7 is one operating system but it's not all the POSIX file system

Reply:
Right, right...